internal/testserver/datastore/mysql.go

```diff
@@ -0,0 +1,111 @@
+package datastore
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+
+	"github.com/authzed/spicedb/internal/datastore"
+	"github.com/authzed/spicedb/internal/datastore/mysql/migrations"
+	"github.com/authzed/spicedb/pkg/migrate"
+	"github.com/authzed/spicedb/pkg/secrets"
+)
+
+var mysqlContainerRunOpts = &dockertest.RunOptions{
+	Repository: "mysql",
+	Tag:        "5",
+	Env:        []string{"MYSQL_ROOT_PASSWORD=secret"},
+	// increase max connections (default 151) to accommodate tests using the same docker container
+	Cmd: []string{"--max-connections=500"},
+}
+
+const (
+	mysqlPort    = 3306
+	defaultCreds = "root:secret"
+	testDBPrefix = "spicedb_test_"
+)
+
+type mysqlBuilder struct {
+	db      *sql.DB
+	creds   string
+	port    string
+	options MySQLBuilderOptions
+}
+
+type MySQLBuilderOptions struct {
+	Prefix  string
+	Migrate bool
+}
+
+// NewMySQLBuilder returns a TestDatastoreBuilder for the mysql driver
+// backed by a MySQL instance
+func NewMySQLBuilder(t testing.TB) TestDatastoreBuilder {
```

Comment:
Driver is named `MySql`; please choose one (probably `MySQL`)

Reply:
addressed in https://github.com/authzed/spicedb/pull/525/commits/94047dae2da9633e1ad619fa51e08da59c48d310